google_api_availability_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+* Add support of namespace property to support Android Gradle Plugin (AGP) 8
```

Comment:
nit: entries in the changelog should use present tense indicative for verbs, with "this version" as an implied subject. For example, "Adds cool new feature.", not "Add" or "Added".
```suggestion
* Adds support for the namespace property to support Android Gradle Plugin (AGP) 8.
```